tests/utils/test_sal_on_coco_dets.py

```diff
@@ -0,0 +1,102 @@
+from click.testing import CliRunner
+import os
+import py
+import pytest
+
+from tests import DATA_DIR
+
+from xaitk_saliency.utils.bin.sal_on_coco_dets import sal_on_coco_dets
+
+from importlib.util import find_spec
+deps = ['kwcoco']
+specs = [find_spec(dep) for dep in deps]
+usable = all([spec is not None for spec in specs])
+
+dets_file = os.path.join(DATA_DIR, 'test_dets.json')
+config_file = os.path.join(DATA_DIR, 'config.json')
+
+
+class TestSalOnCocoDets:
+    """
+    These tests make use of the `tmpdir` fixture from `pytest`. Find more
+    information here: https://docs.pytest.org/en/6.2.x/tmpdir.html
+    """
+
+    @pytest.mark.skipif(not usable, reason="Extra 'xaitk-saliency[tools]' not installed.")
```

Comment:
If this is on all of the test functions, this decorator can be applied to the class to bulk skip.